marlowe-dashboard-client/src/MainFrame/Lenses.purs

```diff
@@ -15,25 +15,25 @@ import Data.Lens.Record (prop)
 import Data.Symbol (SProxy(..))
 import MainFrame.Types (State, WebSocketStatus)
 import Marlowe.Semantics (Slot)
-import Pickup.Types (State) as Pickup
 import Play.Types (State) as Play
 import Toast.Types (State) as Toast
+import Welcome.Types (State) as Welcome
 
 _webSocketStatus :: Lens' State WebSocketStatus
 _webSocketStatus = prop (SProxy :: SProxy "webSocketStatus")
 
 _currentSlot :: Lens' State Slot
 _currentSlot = prop (SProxy :: SProxy "currentSlot")
 
-_subState :: Lens' State (Either Pickup.State Play.State)
+_subState :: Lens' State (Either Welcome.State Play.State)
 _subState = prop (SProxy :: SProxy "subState")
 
 _toast :: Lens' State Toast.State
 _toast = prop (SProxy :: SProxy "toast")
 
 ------------------------------------------------------------
-_pickupState :: Traversal' State Pickup.State
-_pickupState = _subState <<< _Left
+_welcomeState :: Traversal' State Welcome.State
+_welcomeState = _subState <<< _Left
```

Comment:
I like this renaming :) 

Maybe it make sense to rename _playState into _runState in a following PR 🤔

Reply:
Yes, I've already made a start on precisely that. I was thinking of `_dashboardState`, and of combining it with the `ContractHome` stuff. Because the new designs explicitly call that screen the "dashboard", and because contacts and contract setup stuff will be on a modal instead of in their own screen.

We could go with `_runState` instead... :thinking: What's your preference? I think I've a slight preference for "dashboard", because the whole thing is now being called "run" (and at some point we should rename the directory), so we would effectively have `Run = Welcome | Dashboard`.

Reply:
Mmhmhm, I like both options instead of play. My only concern there is that Dashboard would be what is currently the ContractHome, but there are other screens inside that state.

Reply:
Yes, but there won't be other screens there after the next redesign.* That's why I said that contacts and contract setup stuff will be in the new sidebar modal thing in my previous comment (and not in separate screens, as now) - but I didn't explain what the thinking was there, sorry.

* There will be a screen for each contract inside the dashboard, but that feels right to me. The breadcrumb will be "Dashboard > My contract name".